asserts/account_key.go

```diff
@@ -0,0 +1,147 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package asserts
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"golang.org/x/crypto/openpgp/packet"
+)
+
+// AccountKey holds an account-key assertion.
+type AccountKey struct {
+	AssertionBase
+	since     time.Time
+	until     time.Time
+	publicKey *packet.PublicKey
+}
+
+// AccountID returns the account-id of this account-key.
+func (ak *AccountKey) AccountID() string {
+	return ak.Header("account-id")
+}
+
+// Since returns the valid since date of this account-key.
```

Comment:
How about:

    // Since returns the time when the account key starts being valid.

and

    // Until returns the time when the account key stops being valid.

?